ngraph_onnx/onnx_importer/ops_bridge.py

```diff
@@ -73,6 +74,14 @@ def Ceil(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> Ngra
     return ng.ceiling(ng_inputs[0])
 
 
+def Cast(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> NgraphNode
+    """Limit input tensor values within specified interval."""
+    data = ng_inputs[0]
+    new_type = onnx.mapping.TENSOR_TYPE_TO_NP_TYPE[onnx_node.get_attribute_value('to')]
```

Comment:
This line does a lot at once, if it fails it may be difficult to debug. Separate out: 
`old_type = onnx_node.get_attribute_value('to')`